changelogs/fragments/5385-search_s-based-_is_value_present.yaml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - ldap_attrs - fixes issue 977 by ignoring the ``{x}`` prefix on attribute values (https://github.com/ansible-collections/community.general/pull/5385).
```

We usually place the link for the original issue in the `(...)` by the end.
```suggestion
  - ldap_attrs - fix issue by ignoring the ``{x}`` prefix on attribute values (https://github.com/ansible-collections/community.general/issues/977, https://github.com/ansible-collections/community.general/pull/5385).
```
These fragments end up in changelogs like this:
https://github.com/ansible-collections/community.general/blob/stable-5/CHANGELOG.rst#v5-7-0

---

Other than this, LGTM

---

Instead of just saying "fix issue" it should at least contain 1-2 words on what the issue is. In this case, it's related to ordering, so maybe "fix ordering issue"?